packages/oui-message/README.md

```diff
@@ -4,55 +4,44 @@
 
 ## Usage
 
-### Success
-
-```html:preview
-<oui-message message="Nullam a pellentesque nisl. Aliquam erat volutpat. Sed sed ornare diam, ut sollicitudin mauris. Quisque tincidunt dapibus augue, et tincidunt arcu sagittis et. Nam sed aliquet tellus, vel tincidunt urna. Aliquam a placerat enim. Donec condimentum sed nibh a egestas. Maecenas at leo et magna lobortis pretium. Praesent vitae fermentum dui. Interdum et malesuada fames ac ante ipsum primis in faucibus." type="success"></oui-message>
+```html
+<oui-message 
+    type="info|warning|error"
```

Comment:
(optional) We will need success.
- Now, with eventually a deprecated phase (otherwise all the current success message will have to be replaced during the integration step)
- Later (even if it's no in the Message Spec anymore), the temporary success modal has the same UI than a success message (to confirm).

Reply:
yeah, I agree !